soroban-env-host/src/host.rs

```diff
@@ -1891,14 +1891,14 @@ impl VmCallerCheckedEnv for Host {
         self.create_contract_with_id_preimage(code, id_preimage)
     }
 
-    fn create_token_from_contract(
+    fn create_stellar_asset_from_contract(
```

Comment:
The name of this function is quite interesting, because it sounds like something is being created on classic from a contract that already exists, but actually this will create a contract.

Reply:
I removed this host function from the most recent iteration. We can add it back it once we've decided how it'll work.